truss/cli/cli.py

```diff
@@ -410,7 +411,11 @@ def chains():
     """Subcommands for truss chains"""
 
 
-def _make_chains_curl_snippet(run_remote_url: str) -> str:
+def _make_chains_curl_snippet(run_remote_url: str, environment: Optional[str]) -> str:
+    if environment:
+        run_remote_url = re.sub(
```

Comment:
this feels a little hacky & is harder to read than a simple construction, is there a way we can construct this more directly rather than using regex subsitution?

Reply:
went with truncating the string and adding `environment/{environment}/run_remote` to the end